assets/js/base/components/tabs/index.tsx

```diff
@@ -10,7 +10,50 @@ import { useTabState, Tab, TabList, TabPanel } from 'reakit/Tab';
  */
 import './style.scss';
 
-const Tabs = ( {
+export interface TabsProps {
+	/**
+	 * Component wrapper classname
+	 */
+	className?: string;
+	/**
+	 * Event handler triggered when a tab is selected
+	 */
+	onSelect?: ( tabName: string ) => void;
+	/**
+	 * Array of tab objects
+	 */
+	tabs: Array< {
+		name: string;
+		title: string;
+		content: JSX.Element;
+		ariaLabel?: string;
+	} >;
+	/**
+	 * Classname to be applied to the active tab
+	 */
+	activeClass?: string;
+	/**
+	 * Name of the tab to be selected by default
+	 */
+	initialTabName?: string | undefined;
+	/**
+	 * Aria label for the tablist
+	 */
+	ariaLabel?: string;
+	/**
+	 * Instance ID for the component
+	 */
+	instanceId: number;
+	/**
+	 * ID for the component
+	 */
+	id?: string;
+}
+
+/**
+ * Exporting the component for Storybook. Use the default export instead.
+ */
+export const __Tabs = ( {
```

Comment:
I thought `__TabsWithoutInstanceId` would be a better name for this, it's a little more descriptive. What do you think?